docs/trust-list-specification.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 4
+---
+
+# Trust List Specification
+
+## Introduction
+In order for data to be exchanged between different COVID-19 credential standards, there needs to be a means of unifying the trust lists of each respective specification. A Global Trust Network has the opportunity to bring existing and new local and regional trust networks together in a way that respects their designs and sovereignty while contributing to real interoperability among all of them.
```

Comment:
```suggestion
In order for data to be exchanged between different health credential standards, there needs to be a means of unifying the trust lists of each respective specification. A Global Trust Network has the opportunity to bring existing and new local and regional trust networks together in a way that respects their designs and sovereignty while contributing to real interoperability among all of them.
```

Reply:
Mention of "COVID-19" was removed from the Introduction here in [Neelima's most recent commit](https://github.com/dvci/cc-pki/pull/9/commits/8b2e2c8c6967ee1be7613727e5717f49685c1494#diff-c3cc675368d2ab3fcd9e3750a5e781ce5cb26bed90ccc252e4f1ed0e9c9c9e7eR8).